### DISCUSSION TITLE
An announcements/blog-type page for communicating to users of your ecosystem

### DISCUSSION THREAD

**Discussion by byeuji** [sct/overseerr#3742] [Feature Requests]
I'm fairly new to Overseerr, but been using Sonarr for a long time. This is an excellent piece of software (much better than the google form I used to use),

I think it would be really helpful to have some kind of an announcements page or blog feed where I can post, and users can choose to be notified when posted, so that I can easily communicate with all users of my ecosystem. 

For instance, I have certain ground rules on what's in and what's not, and folks can forget these things over time, or when I add folks to the ecosystem it can be overwhelming to dump all the rules on them at the start. 
Additionally, if the system needs to go down because I'm making updates or something, it would be nice to have a single place where they can go to see why they're not able to access Plex, etc.

I searched for a similar feature request but didn't see one listed. Apologies if this is a duplicate.

**Comment by foughtgeoduck** [comment]
I would also love this, easy way to tell users that Plex or other systems are down for maintenance etc.
